packages/quantic/force-app/main/default/lwc/quanticSmartSnippetAnswer/quanticSmartSnippetAnswer.js

```diff
@@ -0,0 +1,81 @@
+import {LinkUtils} from 'c/quanticUtils';
```

Comment:
is this in another branch?

Reply:
yes, it's here: https://github.com/coveo/ui-kit/pull/2707